src/Data/Abstracttile.java

```diff
@@ -0,0 +1,40 @@
+package Data;
+
+public class Abstracttile {
```

Comment:
Klassennamen in Java immer camel-case `AbstractTile` -> Verbessert Lesbarkeit deutlich; ist Convention.